src/controllers/measure.controller.js

```diff
@@ -18,7 +18,9 @@ const getMeasureResults = async (req, res, next) => {
 
 const getDailyMeasureResults = async (_req, res, next) => {
   try {
-    const patientResults = await dao.findMembers({});
+    const measurementYear = _req.query.measurementYear ? parseInt(_req.query.measurementYear, 10)
```

Comment:
One of the eslint rules being applied to this project is that if you have an unused variable you can put an underscore in front of it and it will stop complaining. So feel free to remove that since it's being used now.